### PR TITLE
add more countries available for legal representative

### PR DESCRIPTION
**Pull request by Epimodev**
This PR impacts the company onboarding, registration step only for german account.  
It adds the possibility to set any country for legal representative except USA.

### CODE REVIEW

File: clients/onboarding/src/pages/v2company/OnboardingCompanyRegistration.tsx [hunk 1]
```diff
@@ -45,6 +42,8 @@ import {
   validateRequired,
 } from "../../utils/validation";
 
+const countryItems = Lazy(() => allCountriesItems.filter(item => item.cca3 !== "USA"));
```
Comment (bloodyowl) [inline clients/onboarding/src/pages/v2company/OnboardingCompanyRegistration.tsx:45]: can you just add a small comment explaining the filter?

**Review by bloodyowl — APPROVED** [review]

File: clients/onboarding/src/pages/v2company/OnboardingCompanyRegistration.tsx (hunk 1 above)
Comment (Epimodev) [reply clients/onboarding/src/pages/v2company/OnboardingCompanyRegistration.tsx:45]: you're right will ask the product team

Comment (Epimodev) [reply clients/onboarding/src/pages/v2company/OnboardingCompanyRegistration.tsx:45]: I didn't have answers from product team and it seems there are in holidays. Waiting next week for an answer before merging this

Comment (Epimodev) [reply clients/onboarding/src/pages/v2company/OnboardingCompanyRegistration.tsx:47]: I just received an answer and add a comment here: 2c103dda19cb6b437fa890cac677fafce947c8fd

**Pull request merged**